src/command_modules/azure-cli-network/azure/cli/command_modules/network/_help.py

```diff
@@ -2058,34 +2129,46 @@
           short-summary: Name of resource group. Required when enabling new regions.
           long-summary: When a previously disabled region is enabled to use Network Watcher, a
             Network Watcher resource will be created in this resource group.
+    examples:
+        - name: Configure Network Watcher in a region
+          text: az network watcher configure --resource-group NetworkWatcherRG  --locations westus --enabled true
 """
 
 helps['network watcher troubleshooting'] = """
     type: group
-    short-summary: (PREVIEW) Manage Network Watcher troubleshooting sessions.
+    short-summary: Manage Network Watcher troubleshooting sessions.
+    long-summary: For more information on configuring troubleshooting, visit https://docs.microsoft.com/en-us/azure/network-watcher/network-watcher-troubleshoot-manage-cli
 """
 
 helps['network watcher troubleshooting start'] = """
     type: command
     short-summary: Troubleshoot issues with VPN connections or gateway connectivity.
+    long-summary: For more information on configuring troubleshooting, visit https://docs.microsoft.com/en-us/azure/network-watcher/network-watcher-troubleshoot-manage-cli
     parameters:
         - name: --resource-type -t
           short-summary: The type of target resource to troubleshoot, if resource ID is not specified.
         - name: --storage-account
           short-summary: Name or ID of the storage account in which to store the troubleshooting results.
         - name: --storage-path
           short-summary: Fully qualified URI to the storage blob container in which to store the troubleshooting results.
+    examples:
+        - name: Start a troubleshooting operation
+          text: az network watcher troubleshooting start --resource-group resourceGroupName --resource resourceName --resource-type {vnetGateway/vpnConnection} --storage-account storageAccountName  --storage-path https://{storageAccountName}.blob.core.windows.net/{containerName}
 """
 
 helps['network watcher troubleshooting show'] = """
     type: command
     short-summary: Get the results of the last troubleshooting operation.
+    long-summary: For more information on troubleshooting, visit https://docs.microsoft.com/en-us/azure/network-watcher/network-watcher-troubleshoot-manage-cli
+    examples:
+        - name: Show the results or status of a troubleshooting operation
+          text: az network watcher troubleshooting show  --resource resourcename --resource-group resourcegroupname --resource-type vnetGateway
```

Comment:
Not sure of the value of the example. It only demonstrates the required arguments.